tts/scripts/tts_node.py

```diff
@@ -59,6 +60,8 @@
 
 from sound_play.libsoundplay import SoundClient
 
+# Output files will be symlinked to this path
+SOUND_FILE_PATH = os.path.join(os.sep, 'tmp', 'tts_sound_file')
```

Comment:
Make this platform independent: https://docs.python.org/2/library/tempfile.html#tempfile.tempdir